ext/config.w32

```diff
@@ -0,0 +1,12 @@
+// vim:ft=javascript
```

Comment:
Please remove this.

Reply:
Resolved by https://github.com/maxmind/MaxMind-DB-Reader-php/pull/115/commits/03c7b2f2ccc1555110f83bc077c462ac6ee463e3